storagenode/storagenodedb/satellites.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"github.com/zeebo/errs"
+)
+
+// ErrSatellites represents errors from the satellites database.
```

Comment:
I think the comment has to exactly match the exported variable- so should be `ErrSatellitesDB represents.."

Reply:
Oops! Fixed :)